Tools/WinMLRunner/WinMLRunner.sln

```diff
@@ -10,38 +10,62 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "WinMLRunnerTest", "..\..\Te
 		{81EA9CC6-8A26-4583-B1A4-84740EF815C8} = {81EA9CC6-8A26-4583-B1A4-84740EF815C8}
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ClassLibrary1", "..\ClassLibrary1\ClassLibrary1.csproj", "{12E5A5A7-E32C-4D2E-84DC-E937BE0A9DA8}"
```

Comment:
Was a Csharp project supposed to be added to this solution in the PR with the name: ClassLibrary1?